plugins/bernard.md

```diff
@@ -16,19 +16,22 @@ This plugin provides you tools for both sending commands to and consuming from a
 
 ### Remote execution
 
-To send a command to a queue, simply pass the middleware to the Command Bus. Currently only commands implementing `League\Tactician\Bernard\QueueableCommand` can be sent to the queue, others will be passed to the next middleware in the chain.
+To send a command to a queue, simply pass the middleware to the Command Bus. You can implement `League\Tactician\Bernard\QueueableCommand` to mark a command as queueable. (Alternatively you can implement both `Bernard\Message` and `League\Tactician\Command`) Others will be passed to the next middleware in the chain.
```

Comment:
Phrasing nitpick here but maybe this:

```"To send a command to a queue, simply pass the middleware to the Command Bus" ```

Should be something like:

```To setup queueing on the client side, simply pass the QueueMiddleware to the Command Bus. After that you queue commands by implementing the...```

Right now, it could be read that everytime you want to queue a command, you should pass the middleware to the middleware which would be very difficult indeed :sweat_smile: